sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs

```diff
@@ -205,6 +211,38 @@ public async Task<HttpResponseMessage> ConvertAsync(HttpRequestMessage input, Ca
             return (T)blob.BlobClient;
         }
 
+        private static ParameterBindingData CreateBindingData(BlobAttribute blobAttribute)
+        {
+            var blobPath = BlobPath.ParseAndValidate(blobAttribute.BlobPath);
+            var connectionName = blobAttribute.Connection ?? Constants.DefaultAzureStorageConnectionName;
+
+            dynamic blobDetails = new ExpandoObject();
+            blobDetails.Connection = connectionName;
+            blobDetails.BlobName = blobPath.BlobName;
+            blobDetails.ContainerName = blobPath.ContainerName;
+
+            return CreateParameterBindingData(blobDetails);
+        }
+
+        private static ParameterBindingData ConvertToBindingData(BlobBaseClient input, BlobTriggerAttribute blobAttribute)
+        {
+            var connectionName = blobAttribute.Connection ?? Constants.DefaultAzureStorageConnectionName;
+
+            dynamic blobDetails = new ExpandoObject();
+            blobDetails.Connection = connectionName;
+            blobDetails.BlobName = input.Name;
+            blobDetails.ContainerName = input.BlobContainerName;
+
+            return CreateParameterBindingData(blobDetails);
+        }
+
+        private static ParameterBindingData CreateParameterBindingData(object blobDetails)
+        {
+            var assemblyName = AssemblyName.GetAssemblyName(Assembly.GetExecutingAssembly().Location).Name;
```

Comment:
thinking for Source, we should use the same extension name we use here https://github.com/Azure/azure-sdk-for-net/blob/cb3c61938d63c63313ae0fb8de876f7a42bdaf81/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs#L30, which the platform uses elsewhere when referencing extensions.

Reply:
Is there a way to reference this or should I hard code the string in a similar way? i.e. `source = "AzureStorageBlobs"`